cmd/algorand-indexer/daemon.go

```diff
@@ -141,6 +145,7 @@ func init() {
 	daemonCmd.Flags().StringVarP(&metricsMode, "metrics-mode", "", "OFF", "configure the /metrics endpoint to [ON, OFF, VERBOSE]")
 	daemonCmd.Flags().DurationVarP(&writeTimeout, "write-timeout", "", 30*time.Second, "set the maximum duration to wait before timing out writes to a http response, breaking connection")
 	daemonCmd.Flags().DurationVarP(&readTimeout, "read-timeout", "", 5*time.Second, "set the maximum duration for reading the entire request")
+	daemonCmd.Flags().Uint32VarP(&maxConn, "max-conn", "", 4, "set the maximum connections allowed in the connection pool")
```

Comment:
4 is the default? Seems small, I think making it optional and setting it to 0 would be better.

Please update the readme, we should mention that we recommend people should set this to a number smaller than their Postgres server's configuration.

Reply:
4 is the default number of the postgres pool but we can make it optional.